<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActionWizardWindow</class>
 <widget class="QDialog" name="ActionWizardWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>470</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>470</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>DeepOrder</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
	background-color: #EFEFEF;
}</string>
  </property>
  <widget class="QWidget" name="mainwindow" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>470</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>.AppleSystemUIFont</family>
     <bold>false</bold>
    </font>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>480</width>
      <height>450</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page1">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QFrame" name="frame1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>480</width>
        <height>450</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame {
	background: white;
	border:1px solid black;
	border-radius: 5px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <widget class="QPushButton" name="button_next1">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>410</y>
         <width>80</width>
         <height>30</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
         <bold>false</bold>
         <hintingpreference>PreferVerticalHinting</hintingpreference>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border-radius: 5px;
	color:black;
    border: 1px solid #8f8f91;
    background-color: darkgray;
    padding: 2px 4px;
}</string>
       </property>
       <property name="text">
        <string>다음</string>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::InsertText"/>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_title1">
       <property name="geometry">
        <rect>
         <x>1</x>
         <y>1</y>
         <width>478</width>
         <height>60</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>128</horstretch>
         <verstretch>60</verstretch>
        </sizepolicy>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>20</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
	background-color: deepskyblue;
	color: white;
	border: none;
	border-top-left-radius :5px;
	border-top-right-radius : 5px;
	border-bottom-left-radius : 0px;
	border-bottom-right-radius : 0px;
}</string>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
       <property name="text">
        <string>Action Name</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_preview1">
       <property name="geometry">
        <rect>
         <x>65</x>
         <y>130</y>
         <width>350</width>
         <height>250</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
	color: black;
	background-color: #EFEFEF;
	border-radius: 0px;
	border: 1px solid #8f8f91;
}</string>
       </property>
       <property name="text">
        <string>이곳을 클릭하여
이미지를 선택하세요</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_step1">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>80</y>
         <width>80</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>15</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	border-top-left-radius :15px;
	border-top-right-radius : 0px;
	border-bottom-left-radius :15px;
	border-bottom-right-radius : 0px;
	border: none;
	background-color: deepskyblue;
	color: white;
}</string>
       </property>
       <property name="text">
        <string>STEP 1</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_step1_text">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>80</y>
         <width>280</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	border-top-left-radius :0px;
	border-top-right-radius : 15px;
	border-bottom-left-radius :0px;
	border-bottom-right-radius : 15px;
	border: none;
	background-color: #EFEFEF;
	color: black;
}</string>
       </property>
       <property name="text">
        <string>주문 팝업의 스크린샷을 찍어 이미지를 첨부하세요</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="button_cancel">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>410</y>
         <width>80</width>
         <height>30</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
         <bold>false</bold>
         <hintingpreference>PreferVerticalHinting</hintingpreference>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border-radius: 5px;
	color:black;
    border: 1px solid #8f8f91;
    background-color: #f0f0f0;
    padding: 2px 4px;
}</string>
       </property>
       <property name="text">
        <string>취소</string>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::InsertText"/>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page2">
     <widget class="QFrame" name="frame2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>480</width>
        <height>450</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame {
	background: white;
	border:1px solid black;
	border-radius: 5px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <widget class="QPushButton" name="button_next2">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>410</y>
         <width>80</width>
         <height>30</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
         <bold>false</bold>
         <hintingpreference>PreferVerticalHinting</hintingpreference>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border-radius: 5px;
	color:black;
    border: 1px solid #8f8f91;
    background-color: darkgray;
    padding: 2px 4px;
}</string>
       </property>
       <property name="text">
        <string>다음</string>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::InsertText"/>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_title2">
       <property name="geometry">
        <rect>
         <x>1</x>
         <y>1</y>
         <width>478</width>
         <height>60</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>128</horstretch>
         <verstretch>60</verstretch>
        </sizepolicy>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>20</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
	background-color: deepskyblue;
	color: white;
	border: none;
	border-top-left-radius :5px;
	border-top-right-radius : 5px;
	border-bottom-left-radius : 0px;
	border-bottom-right-radius : 0px;
}</string>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
       <property name="text">
        <string>Action Name</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="button_prev1">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>410</y>
         <width>80</width>
         <height>30</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
         <bold>false</bold>
         <hintingpreference>PreferVerticalHinting</hintingpreference>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border-radius: 5px;
	color:black;
    border: 1px solid #8f8f91;
    background-color: #f0f0f0;
    padding: 2px 4px;
}</string>
       </property>
       <property name="text">
        <string>이전</string>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::InsertText"/>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_preview2">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>130</y>
         <width>340</width>
         <height>240</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
	color: black;
	background-color: #EFEFEF;
	border-radius: 0px;
	border: 1px solid #8f8f91;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_step2">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>80</y>
         <width>80</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>15</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	border-top-left-radius :15px;
	border-top-right-radius : 0px;
	border-bottom-left-radius :15px;
	border-bottom-right-radius : 0px;
	border: none;
	background-color: deepskyblue;
	color: white;
}</string>
       </property>
       <property name="text">
        <string>STEP 2</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_step2_text">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>80</y>
         <width>240</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	border-top-left-radius :0px;
	border-top-right-radius : 15px;
	border-bottom-left-radius :0px;
	border-bottom-right-radius : 15px;
	border: none;
	background-color: #EFEFEF;
	color: black;
}</string>
       </property>
       <property name="text">
        <string>시간조정 요소의 위치를 지정해주세요</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_tip1">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>370</y>
         <width>340</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border:none;
color:black;</string>
       </property>
       <property name="text">
        <string>이미지를 클릭하여 위치를 지정하세요</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignHCenter</set>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page3">
     <widget class="QFrame" name="frame3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>480</width>
        <height>450</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame {
	background: white;
	border:1px solid black;
	border-radius: 5px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <widget class="QPushButton" name="button_save">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>410</y>
         <width>80</width>
         <height>30</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
         <bold>false</bold>
         <hintingpreference>PreferVerticalHinting</hintingpreference>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border-radius: 5px;
	color:black;
    border: 1px solid #8f8f91;
    background-color: darkgray;
    padding: 2px 4px;
}</string>
       </property>
       <property name="text">
        <string>완료</string>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::InsertText"/>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_title3">
       <property name="geometry">
        <rect>
         <x>1</x>
         <y>1</y>
         <width>478</width>
         <height>60</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>128</horstretch>
         <verstretch>60</verstretch>
        </sizepolicy>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>20</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
	background-color: deepskyblue;
	color: white;
	border: none;
	border-top-left-radius :5px;
	border-top-right-radius : 5px;
	border-bottom-left-radius : 0px;
	border-bottom-right-radius : 0px;
}</string>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
       <property name="text">
        <string>Action Name</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="button_prev2">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>410</y>
         <width>80</width>
         <height>30</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
         <bold>false</bold>
         <hintingpreference>PreferVerticalHinting</hintingpreference>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border-radius: 5px;
	color:black;
    border: 1px solid #8f8f91;
    background-color: #EFEFEF;
    padding: 2px 4px;
}</string>
       </property>
       <property name="text">
        <string>이전</string>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::InsertText"/>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_preview3">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>130</y>
         <width>340</width>
         <height>240</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
	color: black;
	background-color: #EFEFEF;
	border-radius: 0px;
	border: 1px solid #8f8f91;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_step3">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>80</y>
         <width>80</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>15</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	border-top-left-radius :15px;
	border-top-right-radius : 0px;
	border-bottom-left-radius :15px;
	border-bottom-right-radius : 0px;
	border: none;
	background-color: deepskyblue;
	color: white;
}</string>
       </property>
       <property name="text">
        <string>STEP 3</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_step3_text">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>80</y>
         <width>240</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	border-top-left-radius :0px;
	border-top-right-radius : 15px;
	border-bottom-left-radius :0px;
	border-bottom-right-radius : 15px;
	border: none;
	background-color: #EFEFEF;
	color: black;
}</string>
       </property>
       <property name="text">
        <string>수락/거절 버튼의 위치를 지정해주세요</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_tip2">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>370</y>
         <width>340</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border:none;
color:black;</string>
       </property>
       <property name="text">
        <string>이미지를 클릭하여 위치를 지정하세요</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignHCenter</set>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
